### ISSUE TITLE
Can we/should we capture deleted tweets that initiated or closed a job and delete the job?

### ISSUE THREAD

**Issue by gordonje** [rji-futures-lab/tweets-to-text#11]

**Comment by gordonje** [comment]
I just ran an experiment to see if we would receive a notification for the deletion of a tweet mentioning @TweetsToText. 

We didn't 😞 

I've also posted this question to the [Twitter Developer Forum](https://twittercommunity.com/t/tweet-delete-events-for-mentions-of-the-account-to-which-im-subscribed/116086) to confirm this behavior.

Here's an outline of an alternative approach:

1. When a new mention comes in, check to see if there's an outstanding job for the author of the tweet.
2. If there is an outstanding job, check to see if that tweet is still available.
    - If the tweet is available, treat the mention as final and process the job.
    - If the tweet is not available, create a new job.
3. If there isn't an outstanding job, create a new job.
